pages/about.html

```diff
@@ -4,9 +4,34 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Page</title>
+  <!-- Link to Style Sheet -->
+  <link href="styles/style.css" rel="stylesheet">
 </head>
 <body>
   
+  <header>
+      <nav>
+        <h4>Menu</h4>
+        <a href="/pages/index.html">Index</a>
+        <a href="/pages/portfolio.html">Portfolio</a>
+      </nav>
```

Comment:
I just want to point out here that the light text on light background is *hard* to read.